src/core/init/duration_updater.ts

```diff
@@ -42,7 +45,11 @@ export default function DurationUpdater(
 ) : Observable<never> {
   return fromEvent(manifest, "manifestUpdate").pipe(
     startWith(null),
-    tap(() => {
+    switchMap(() => {
```

Comment:
Is it only in initialization cases? If so why not just using combineLatest or something like that?

Reply:
What is strange is that it seems not to happen in initialization cases. We've noticed it when playing multiperiod live content, when reaching the live edge. I'm investigating to find the cause.

Reply:
For that type of cases, i consider the switchMap to be more appropriate, as it cancels last subscription of inner observables.